tests/ctst/common/common.ts

```diff
@@ -66,3 +78,32 @@ Given('an object {string} that {string}',
             ));
         }
     });
+
+Then('kafka consumed messages should not take too much place on disk',
+    async function (this: Zenko) {
+        const notToCheckTopics = ['oplog', 'dead-letter'];
```

Comment:
"topics not to check" → `topicsNotToCheck` (or `topicsToIgnore`)
or the other way with an adjective : `ignoredTopics`